admin_manual/appliance/Clamav.rst

```diff
@@ -78,6 +78,11 @@ If you want to refresh the ClamAV database manually, follow these steps:
   # Start the refresh process again with this command:
   sudo freshclam
 
+..note::
+
+	- When the app is just enabled (without configuration or with wrong configuration) it will strictly reject **all** uploads for the whole instance.
```

Comment:
Proposal:
``When the app is just enabled (without configuration or with wrong configuration) it`` -->
``When the app is enabled only, means without configuration or with a wrong configuration it``

Reply:
How about:

```rst
..note::

  - When the app is enabled — but is not configured or has an incorrect configuration — it will reject **all** uploads for the entire instance.
```